### PR TITLE
vim 8.0.0600

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/vim.rb [hunk 1]
```diff
@@ -82,13 +81,18 @@ def install
...
-    if build.with? "lua"
-      opts << "--enable-luainterp"
-    end
-
-    if build.with? "luajit"
-      opts << "--with-luajit"
+    if build.with?("lua") || build.with?("luajit")
+      ENV["LUA_PREFIX"] = HOMEBREW_PREFIX
       opts << "--enable-luainterp"
+      opts << "--with-luajit" if build.with? "luajit"
```
Comment (ilovezfs) [inline Formula/vim.rb]: does --with-luajit not imply --enable-luainterp automatically?

Comment (DomT4) [inline Formula/vim.rb]: Hilariously, no. If you don't pass `--enable-luainterp` first it'll ignore that you ever mentioned `luajit`.

Comment (JCount) [inline Formula/vim.rb]: ......................................

**Comment by DomT4** [comment]
Are folks happy on this?

**Comment by ilovezfs** [comment]
:cookie:

**Pull request closed without merging**